test/skimNtuple2017.cpp

```diff
@@ -3411,8 +3825,19 @@ int main (int argc, char** argv)
 
             if (TMath::Abs(tlv_jet.Eta()) < 4.7)
             {
+             
               theSmallTree.m_BDT_HT20 += tlv_jet.Pt() ;
+
               if (DEBUG) cout << " ---> Jet " << iJet << " - pt: " << tlv_jet.Pt() << " - HT: " << theSmallTree.m_BDT_HT20 << endl;
+              pair <vector <double>, vector<double>> unc_updown = getJetUpDown(iJet, N_jecSources, theBigTree);
+              // build shifted jet
+              for (int isource = 0; isource < N_jecSources; isource++)
+              {
+                TLorentzVector tlv_jetup   = getShiftedJet(tlv_jet, +1., unc_updown.first[isource]);
+                TLorentzVector tlv_jetdown = getShiftedJet(tlv_jet, -1., unc_updown.second[isource]);
```

Comment:
shouldn't we check here that these shifted jets have pT > 20 GeV?

Reply:
uhm I'm not sure, but I can implement that

Reply:
done, they all need to pass the PT > 20 GeV requirement